scripts/publish_grok46_hub.py

```
@@ -0,0 +1,608 @@
+    notes = sorted(src.glob("NOTES-r*.md"))
+    records = 0
+    bytes_ = 0
+    labels = []
+    desired_batch_names = {batch.name for batch in batches}
+    for existing in raw.glob("batch-r*.jsonl"):
+        if existing.name in desired_batch_names:
+            continue
+        if existing.is_file() or existing.is_symlink():
+            existing.unlink()
+        else:
+            raise SystemExit(f"unsafe non-file snapshot payload: {existing}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)